Spacing for Events Page

natashasthilaire opened the pull request:
### About
This PR adjusts the spacing for the share button on mobile and adds spacing between content and the footer for `/events` 

### Test Instructions
1. Go to https://www.christfellowship.church/events/hear-from-coach-tom
2. Visit `/events/hear-from-coach-tom` on the deployment link and note the space between the card and the footer
3. Use the inspect tool to view both sites on mobile and note the additional space under the Share button on mobile

### Screenshots
<img width="1176" alt="Screenshot 2023-07-25 at 3 02 09 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/9b565f8c-14c9-483a-a87b-550367418fe4">

<img width="339" alt="Screenshot 2023-07-25 at 2 56 29 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/f7676bf7-8787-4546-ab99-a51ad45ed4dd">

### Related Tickets
[CFDP-2578](https://christfellowshipchurch.atlassian.net/browse/CFDP-2578)

leoalfonso14 reviewed (dismissed):
Looks good!

codingwithlis reviewed (dismissed):
Looks great, nice job!

dzwood reviewed (approved):
I removed the unused imports and updated the `mb` property. Setting a style for a breakpoint affects all the larger breakpoints as well, until another breakpoint is updated. In this case we only needed to change the sizing for `_` and `lg`, if that makes sense.

@natashasthilaire Great job on this 🎉

natashasthilaire commented:
Makes sense thank you

> I removed the unused imports and updated the `mb` property. Setting a style for a breakpoint affects all the larger breakpoints as well, until another breakpoint is updated. In this case we only needed to change the sizing for `_` and `lg`, if that makes sense.
> 
> @natashasthilaire Great job on this 🎉

Makes sense, thank you!

leoalfonso14 reviewed (approved):
Great job!
